modules/special-rules/pages/dauntless.adoc

```diff
@@ -0,0 +1,9 @@
+= Dauntless
+
+Some troops are so disciplined that they never flinch when facing the enemy -- such as the Dark Angels Deathwing, or Tyranid Swarms with Synapse Creatures.
```

Comment:
Should the examples be removed here as well?

Reply:
Yes, I think that would be better. However, some details:

1. I'd like to remove the construction that leads to an avoidable -ing word.
2. Units might be Dauntless because of innate properties of some kind, not just training.

```suggestion
Some troops are so disciplined or relentless that they never flinch in the face of the enemy.

NOTE:  In comparison to _Epic 40,000_, we've separated this ability from parts of the Dark Angels Deathwing and Tyranid Synapse Creatures rules.
```

Reply:
Suggestion added to commit batch.